Favor/Favor/Sources/Scenes/Application/SceneDelegate.swift

```diff
@@ -24,3 +26,33 @@ class SceneDelegate: UIResponder, UIWindowSceneDelegate {
 		self.appCoordinator?.start()
 	}
 }
+
+private extension SceneDelegate {
+  func setupNavigationBarAppearance() {
+    let appearance = UINavigationBarAppearance()
+    let backButtonAppearance = UIBarButtonItemAppearance()
+    
+    let leftArrowImage = UIImage(named: "ic_leftArrow")?
+      .withRenderingMode(.alwaysOriginal)
+      .withAlignmentRectInsets(UIEdgeInsets(top: 0, left: 0, bottom: 10, right: 0))
+    
+    backButtonAppearance.normal.titleTextAttributes = [
+      .foregroundColor: UIColor.clear
```

Comment:
```suggestion
      .foregroundColor: .clear
```

enum 타입 사용할 때 스타일도 정해두면 좋을 것 같아!

여기서는 앞에 클래스명을 생략을 안하고 사용했길래 다른 코드를 봤더니 거기서는 생략하고 사용했더라고..!

Reply:
아 이게..
`titleTextAttributes`는 
```swift
[NSAttributedString.Key: Any] 
```
타입이라서 

`Any` 타입으로 정의되어 있어서 타입 추론을 못하는 것 같아...ㅠㅠ

Reply:
아하..
그런 경우는 어쩔 수 없지..!

그럼 지금 그대로 진행하자!